deploy/examples/create-external-cluster-resources.py

```diff
@@ -1568,7 +1568,12 @@ def getFinalUsedArgs(self):
                 # python treats flag-name as flag_name internally, so converting back to flag-name,
                 # so we can get those values from config file
                 argValue = arg.replace("_", "-")
-                if argValue != "config-file":
+                # do not add the config-file flag and also not add the boolean flags which are set to False
+                # because config.ini file treats boolean flags as True always
```

Comment:
Before this fix, are you saying that even though a flag was set to `false`, it was still be interpreted as `true`? That just sounds like bad handling of bools. :)

Reply:
Actually in python args bool value has no argument,

Let's say

Python3 --boolvalue 

We don't need to specify true or false, if we just pass the flag it means the value is set true